demos/serverless/src/handlers.js

```diff
@@ -114,8 +114,8 @@ exports.logs = async (event, context) => {
     putLogEventsInput.sequenceToken = uploadSequence;
   }
   const logEvents = [];
-  if (body.logs.length !== 0) {
-    return response(200, 'application/json', JSON.stringify({}));
+  if (body.logs.length === 0) {
```

Comment:
You can move this check to line 103

```
 if (!body.logs || !body.logs.length || !body.meetingId || !body.attendeeId || !body.appName) {
```

Note that `!body.logs.length` will be true if the value is `0` so does the `if` condition